samples/grpc/local-drone-control-scala/src/main/resources/META-INF/native-image/ch.qos.logback/logback-classic/native-image.properties

```diff
@@ -0,0 +1 @@
+Args = --initialize-at-build-time=ch.qos.logback
```

Comment:
Logging needs to be initialized at build time, because Netty is configured to be build time and has loggers in static fields. Netty 5 switches to being initialized at run time by default, which would make native image builds for applications simpler.